src/Components/MovieDetails.js

```diff
@@ -14,11 +14,17 @@ class MovieDetails extends React.Component {
   }
 
   componentDidMount = () => {
-    fetchSingleMovie(this.props.selectedMovie)
-      .then(data => this.setState({
-        movie: data.movie,
-        isLoading: false
-      }))
+    const singleMovie = fetchSingleMovie(this.props.selectedMovie)
+    const faveMovies = fetchFavoriteMovies();
+
+    Promise.all([singleMovie, faveMovies])
```

Comment:
I wasn't sure how this was going to resolve - but I commend you for sticking with it and getting it up and running in this component as well! Good call yet again with the decision to run this Promise.all in the componentDidMount()